watertap/core/util/scaling.py

```diff
@@ -27,3 +32,10 @@ def transform_property_constraints(self):
                 for ind, c in con.items():
                     sf = iscale.get_scaling_factor(var[ind], default=1, warning=True)
                     iscale.constraint_scaling_transform(c, sf)
+            else:
+                msg = (
+                    f"One or more property constraints are not scaled. "
+                    "This suggests that the user defined a property in metadata but failed to "
+                    "follow the the naming convention for its constraint: 'eq_' + name in metadata."
+                )
+                _log.warning(msg)
```

Comment:
```suggestion
            else:
                msg = (
                    f"If there was a property constraint written for the variable, {var}, that constraint was not scaled. "
                    "The transform_property_constraints tool expects constraints to have the following naming convention: 'eq_' + '{var_str}'. This suggests that the user may have defined a property in metadata but failed to "
                    "follow the naming convention for its constraint. If there is no property constraint associated with the {var_str} variable, this warning can be ignored."
                )
                _log.warning(msg)
```

Something like this, since in some cases, there is no constraint- just a reference to a parameter.

Reply:
You may have to fix the f-string for it to work. Also we should have some test file for this utility (see `test_initialization` to compare with test for `watertap.core.util.initialization`).